presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigDeployer.java

```diff
@@ -115,11 +115,21 @@ public void deployDefaultConfig(Path configDir, Path dataDir, Path pluginDir)
                 // TODO: Include node.id in install requests
                 updateProperty(configDir.resolve("node.properties"),
                         "node.id", randomUUID().toString());
+                /**
+                 * To prevent conflicts between Presto-Manager configs and
+                 * Presto configs present in {@link #defaultConfig} directory
+                 */
+                Path nodePropertiesPath = configDir.toAbsolutePath().resolve("node.properties");
```

Comment:
I don't understand why this section is needed.  Why do you update properties for the deploying default configs?  shouldn't that follow defaults?